src/main/java/com/google/api/codegen/transformer/SurfaceTransformerContext.java

```diff
@@ -117,6 +107,13 @@ private boolean isSupported(Method method) {
     return methods;
   }
 
+  private boolean isSupported(Method method) {
+    if (!getFeatureConfig().enableStreaming()) {
+      return !method.getRequestStreaming() && !method.getResponseStreaming();
+    }
+    return true;
+  }
```

Comment:
Not sure how worthwhile doing (and/or how does that fit with the design principle of MVVM), but can we just move `isSupported` into the FeatureConfig -- with receiving `Method` as an argument?